dialogueMaker/dialogueMaker.cpp

```diff
@@ -156,6 +156,13 @@ void Game::cleanUp() {
 }
 
 bool Game::gameEnded() {
+  if (PlayerP.CheckIfdied()) {
+    setCurrentScene("bad_ending");      // Change to the "die" scene
+    printCurrentScene();                // Print the "die" scene
+    Game::ResetSaveFile("save.txt");
+    cleanUp();
+  }
```

Comment:
```suggestion
  if (PlayerP.CheckIfdied()) {
    setCurrentScene("bad_ending");      // Change to the "die" scene
    printCurrentScene();                // Print the "die" scene
    Game::ResetSaveFile("save.txt");
    cleanUp();
    return true;
  }
```
forgot to return true